openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+                    required:
+                      - operations
+                      - timebounds
+              required:
+                - transactions
+            example:
+              transactions:
+                - opearations:
+                  - "Y6MF7SMT2a2d6pt3i37Xx9"
+                  - "M8GA7SMT2a2d6pt3i37Xx8"
+                  timebounds: 300
+      responses:
+        '200':
+          description: "Successful response containing a list of base64 transaction xdr strings"
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  transactionxdrs: 
+                    type: array
+                    items:
+                      type: string
+                      description: base64 encoded transaction xdr string
+              example:
+                transactionxdrs:
+                  - "Y6MF7SMT2a2d6pt3i37Xx9"
+                  - "M8GA7SMT2a2d6pt3i37Xx8"
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                    description: Details about the error
+                example:
+                  status: 400
+                  error: Bad operation xdr.
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                example:
+                  status: 500
+                  error: An error occurred while processing this request.
+              example:
+                status: 500
+                error: An error occurred while processing this request.
+  /tss/transactions:
+    post:
+      tags:
+        - TSS
+      summary: submit transactions in bulk to TSS
+      description:
+        Submit a list of transactions to TSS so that it can submit it to the stellar network
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                webhookurl:
+                  type: string
+                  description: The url to which the final transaction results are to be sent
+                transactions:
+                  type: array
+                  items:
+                    type: string
+                    description: base64 encoded transaction xdr to submit to the stellar network
+                feebump:
+                  type: boolean
+                  description: boolean indicating if this transaction should be wrapped in a fee bump transaction
```

Comment:
Would it be equivalent for the client to submit transactions individually to `/tx/create-fee-bump`, and then submit all of them to this endpoint with the `feebump` parameter set to `false`?

Reply:
Yes thats right. Of course, that would add an additional endpoint for clients of TSS to call though before submitting them. I have to add though that in the /tx/create-fee-bump has some additional logic where it blocks certain operation types (which can be configured). Open to changing the api design around these endpoints though. I'm open to removing the feebump boolean for example